Jenkinsfile

```diff
@@ -27,28 +27,20 @@ pipeline {
     stage("Unit Tests") {
       steps {
         dir("$DIRECTORY") {
-          sh "make test"
+          // sh "make test"
```

Comment:
any particular reason for this section to exist instead of just commenting it out?

Reply:
Probably not. I didn't make a change to that in this PR though, because I'm mostly just pulling in the commits from the release-v1.0.1 branch where that change is already made (by me)